bin/spice/pkg/cli/cmd/login.go

```diff
@@ -150,6 +157,51 @@ spice login dremio --username <username> --password <password>
 	},
 }
 
+func dremioAuth(username string, password string) bool {
+
+	type userCredentials struct {
+		UserName string `json:"userName"`
+		Password string `json:"password"`
+	}
+
+	// TODO: Allow user to specify their own dremio host. Needs UX design for how the command should handle.
+	// TODO: Do we ultimately want to use dremio token instead of username/password in the following dremio requests?
+	const dremioHost = "dremio-4mimamg7rdeve.eastus.cloudapp.azure.com:9047"
```

Comment:
Let's remove this hard-coding, ref: https://github.com/spicehq/spiceai/issues/100